keyboards/wilba_tech/wt60_b/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "wilba.tech WT60-B",
+    "url": "https://wilba.tech",
+    "maintainer": "Wilba",
+    "bootloader": "atmel-dfu",
```

Comment:
A bit of a nitpick, but the `bootloader` key for `info.json` files isn't used anymore.

```suggestion
```